packages/web/CONTRIBUTING.md

```diff
@@ -1,45 +1,38 @@
-## NatDS Web
+# NatDS Web
 
-### Natura Design System - Web Components
+Natura Design System - Web Components
 
----
+## Install dependencies
```

Comment:
Before installing dependencies with yarn, make sure the person has installed and is using our node version (10.16.3) and has installed yarn with the same version as we do. Also, wouldn't it be better to put this installing dependencies on the root since we're using yarn workspaces?